Source/JavaScriptCore/inspector/remote/RemoteInspectionTarget.cpp

```diff
@@ -32,21 +32,33 @@
 #include "RemoteInspector.h"
 #include <wtf/RunLoop.h>
 
+#if PLATFORM(COCOA)
+#include <wtf/spi/darwin/OSVariantSPI.h>
+#endif
+
 namespace Inspector {
 
 bool RemoteInspectionTarget::remoteControlAllowed() const
 {
-    return remoteDebuggingAllowed() || hasLocalDebugger();
+    return inspectable() || hasLocalDebugger();
```

Comment:
ideally we'd entirely git rid of `hasLocalDebugger` and have everything just be off of the one `inspectable` flag

Reply:
I don't think we can entirely get rid of it because we need to inform the remote process that relays inspectable information that the process is being debugged in-process (RemoteInspectorCocoa.mm:497). But yeah we can probably get rid of its use here.

Reply:
to be clear this was more of a wishful thinking than something you have to change right now

i do think eventually we should move to a place where `inspectable` is the single source of truth for "can this be inspected" and then some new `inspected` would be the single source of truth for "is this actively being inspected"

Reply:
put another way, we should see if it's possible (eventually) to entirely remove the distinction between "local" vs "remote" inspection, instead only caring about whether or not something is inspected at all (not where it's coming from)